update-sdk/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: "SDK Outdated"
+description: "Reusable workflow for update the SDK packages"
+
+inputs:
+  repository:
+    description: "Github Repository"
+    default: ${{ github.repository }}
+  branch:
+    description: "Branch name"
```

Comment:
We should be more descriptive here, like `The name of the branch to be created`

Reply:
https://github.com/FuelLabs/github-actions/pull/26/commits/30378159fa1e43b65545271ec435ecd01570def9